service/history/taskAllocator.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/log/tag"
 	"github.com/uber/cadence/service/history/shard"
+	ctask "github.com/uber/cadence/service/history/task"
```

Comment:
drop the import alias?

Reply:
the task is used as variable below

Reply:
I see. I use `ctask` in other places for the `common/task` package as `c` stands for `common`.

Reply:
I see. Then I will try to find another alias